packages/coil-extension/src/popup/context/routerContext.tsx

```diff
@@ -12,12 +13,33 @@ interface IRouterContext {
   back: () => void
 }
 
+interface IRouterProvider {
+  storage: Pick<StorageService, 'get'>
+}
+
 // Context
 const RouterContext = createContext({} as IRouterContext)
 
 // Provider
-export const RouterProvider: React.FC = props => {
-  const [route, setRoute] = useState<string>(ROUTES.streaming)
+export const RouterProvider: React.FC<IRouterProvider> = props => {
+  const { storage } = props
+  const userObject = storage.get('user')
+
+  // Determine the default view based on if the user is allowed to tip
```

Comment:
Is there anyway we could have this remember your last used view in dev mode ? 
This makes things kind of annoying for testing